code/modules/clothing/head/helmet.dm

```diff
@@ -591,6 +591,74 @@ GLOBAL_LIST_INIT(allowed_helmet_items, list(
 	desc = "An experimental brain-bucket. A dust ruffle hangs from back. Moderately better at deflecting blunt objects at the cost of humiliation. But who will be laughing at the memorial? Not you, you'll be busy getting medals for your intel work."
 	specialty = "XM12 pattern intel"
 
+/obj/item/clothing/head/helmet/marine/heavy_assault
+	name = "\improper B15 pattern assault helmet"
+	desc = "A robust hunk of metal and rubber designed to go with the B15 heavy assault armor. It has a built in visor which can slide up and down to better protect the wearer."
+	icon_state = "heavyassault"
+	item_state = "heavyassault"
+	flags_atom = FPRINT|CONDUCT|NO_SNOW_TYPE
+	flags_inventory = COVEREYES|COVERMOUTH|BLOCKSHARPOBJ
+	siemens_coefficient = 0.9
+	force = 20
+	specialty = "B15 pattern assault"
+	var/protection_on = FALSE
+	var/base_icon_state
+
+	actions_types = list(/datum/action/item_action/toggle)
+
+/obj/item/clothing/head/helmet/marine/heavy_assault/Initialize()
+	. = ..()
+	base_icon_state = icon_state
+
+/obj/item/clothing/head/helmet/marine/heavy_assault/attack_self(mob/user)
+	..()
+	toggle()
+
+/obj/item/clothing/head/helmet/marine/heavy_assault/verb/toggle()
```

Comment:
why is this a verb? it already has an action attached, it's redundant and we want to limit the amount of verbs we have. turn it into a normal proc and replace usr

Reply:
Don't use a verb like this.
Use an action button with a keybinding if you want it able to be activated by a keypress